src/global_identifiability.jl

```diff
@@ -142,21 +145,50 @@ function check_identifiability(io_equation::P, parameters::Array{P, 1}, p::Float
     return check_identifiability([io_equation], parameters, p; method=method)
 end
 
+#------------------------------------------------------------------------------
+"""
+    assess_global_identifiability(ode::ODE{P}, p::Float64=0.99; var_change=:default, gb_method=:GroebnerBasis) where P <: MPolyElem{fmpq}
+
+Input:
+- `ode` - the ODE model
+- `p` - probability of correctness
+- `var_change` - a policy for variable change (`:default`, `:yes`, `:no`), affects only the runtime
+- `gb_method` - library used for Groebner bases (`GroebnerBasis`, `Singular`)
```

Comment:
Shouldn't it be `:GeobnerBasis` and `:Singular` ?

Reply:
yes, thank you